docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -254,7 +254,7 @@ tasks
         compilerOptions
             .languageVersion
             .set(
-              org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_1_9
+              org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_2_0
```

Comment:
What if we use a variable here? Will it help us with updating the v.list file instead of updating the example?

I mean if it's not the only place where we have this value, we can use a var.

Reply:
Great idea! At the moment this is the only instance of it but by making a it a variable it's much less likely that we will forget to update it in future. Updated!